manager/integration/framework/volume/__init__.py

```diff
@@ -0,0 +1 @@
+from volume.volume import Volume
```

Comment:
let's simplify module naming.

volume/base.py (add abstract and default volume class here)
volume/crd.py
volume/rest.py

then, we use the same pattern for other resources in the future.

Reply:
> (add abstract and default volume class here)

Put abstract and implementation class in the same file causes import error: `failed: ImportError: cannot import name 'AbstractVolume' from partially initialized module 'volumes.base' (most likely due to a circular import) `

Others refined.

Reply:
Interesting, not sure why having this limitation.